docs/receiving/receiving-with-ingest.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Ingest Overview
+---
+
+import CodeTabs from '@theme/CodeTabs';
+import TabItem from '@theme/TabItem';
+
+[Svix Ingest] grants you scalability, security best practices, and
+reliability, all with minimal configuration.
+
+Creating a `Source` will generate an endpoint you can share with a webhook
+provider as a destination for their webhooks.
+
+Ingest supports a variety of signature verification schemes used by webhook
+providers. Supported values for a `Source`'s' `type` include:
+
+- `beehiiv`
+- `brex`
+- `clerk`
+- `github`
+- `guesty`
+- `hubspot`
+- `incidentIo`
+- `lithic`
+- `nash`
+- `pleo`
+- `replicate`
+- `resend`
+- `safebase`
+- `sardine`
+- `shopify`
+- `stripe`
+- `stych`
+- `svix` (that's us!)
+- `zoom`
+
+Additionally there's the option to configure a `Source` as `http` that doesn't
+perform any signature verification at all for cases where the provider has no
+signing scheme, or if _they do_ but it isn't yet supported by Ingest.
+
+:::tip
+Let us know if your webhook provider isn't yet supported so we can add it!
+:::
+
+
+## Create a Source
+
+Getting started with Ingest is a matter of configuring a `Source`.
+
+<CodeTabs>
+<TabItem value="curl">
+
+```sh
+curl https://api.svix.com/ingest/api/v1/source \
+  -X POST \
+  -H "Authorization: Bearer $SVIX_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d @- << EOF
+{
+  "name": "from-github",
+  "description": "receives webhooks to from GitHub",
+  "uid": "from-github",
+  "type": "github",
+  "config": {
+    "secret": "<the signing secret>"
+  }
+}
+EOF
+```
+
+</TabItem>
+</CodeTabs>
+
+## Tell your provider where to send webhooks
+
+The `SourceOut` response will include an `ingestUrl` which is the endpoint you
+give to your provider, telling them where to send their webhooks.
+For GitHub, as is used in this example, this is referred to as the
+_"Payload URL"_.
+
+## Managing incoming messages
+
+Messages received by Ingest are automatically forwarded to a companion Svix
+`Application`.
+
+The `SourceOut` response also includes an `id` field.
+This `id` appears in the `ingestUrl`, but is also used to generate the `Uid`
+for the Svix `Application`. The `Uid` will be: `app_<the source id>`.
+
+Managing this compainion `Application` for the `Source` is how you can
+configure endpoints, check attempts, retry or recover messages and so on.
```

Comment:
Maybe mention what this means in the context of Ingest here, especially for people not very familiar with Svix core? I'm thinking something as simple as

```suggestion
configure endpoints to forward the incoming webhooks to, check attempts, retry or recover messages and so on.
```

could already help some people.